CHANGELOG.md

```diff
@@ -7,6 +7,51 @@ Mapbox welcomes participation and contributions from everyone.
 - `AnnotationManager` no longer conforms to `Observer` and no longer has a `peer` ([#246](https://github.com/mapbox/mapbox-maps-ios/pull/246))
 - `AnnotationSupportableMap` is now internal ([#246](https://github.com/mapbox/mapbox-maps-ios/pull/246))
 
+- #### MapView
+    * Initializer has been change to `public init(frame: CGRect, resourceOptions: ResourceOptions, glyphsRasterizationOptions: GlyphsRasterizationOptions = GlyphsRasterizationOptions.default, styleURI: StyleURI? = .streets)`.
+    * The following 4 functions have been removed: 
```

Comment:
can/should we explain why?

Reply:
+1

I think this is related to the concern I expressed in https://github.com/mapbox/mapbox-maps-ios/pull/250#discussion_r608850732

I don't like the idea of removing these 4 if we don't have a replacement ready for them. I could see that being something that would prevent folks from upgrading to our next beta.

Reply:
This is quite a large change so I understand why. But my understanding of deprecating these were because they were not being used actively

Reply:
@sbma44 just talked to Julian, we are going to revert this for now